spec/requests/providers_spec.rb

```diff
@@ -1116,4 +1116,14 @@ def gen_import_request
       end
     end
   end
+
+  context "OPTIONS /api/providers" do
+    it "returns options for all providers when no query" do
+      run_options(providers_url)
+      expect(response.parsed_body["data"].keys.count).to eq(
+        ExtManagementSystem.leaf_subclasses.select { |ems| !ems.ems_type.blank? }.count
```

Comment:
No, but if a new one will be created I think it will break this test. I will remove the `select`.